packages/core/helper-plugin/src/hooks/tests/useFocusWhenNavigate.test.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable check-file/filename-naming-convention */
+
+import { lightTheme, ThemeProvider, Main } from '@strapi/design-system';
+import { render, renderHook, screen } from '@testing-library/react';
```

Comment:
I think you can do the same suggestion as: https://github.com/strapi/strapi/pull/17992/files#r1321089847